data/operators.csv

```diff
@@ -42,6 +42,7 @@ TIM,ISP,,unsafe,3269,176
 Rogers,ISP,,unsafe,812,198
 Sunrise Communications AG,ISP,,unsafe,6730,203
 Jaguar Network,ISP,signed + filtering,safe,30781,226
+HiNet,ISP,,unsafe,3462,249
```

Comment:
As tested, the status should be safe. Seem like they are filtering.

Reply:
Seems they updated it after I wrote the commit. Changed.